runtime/frequency/src/lib.rs

```diff
@@ -518,6 +615,7 @@ construct_runtime!(
 		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 2,
 		ParachainInfo: parachain_info::{Pallet, Storage, Config} = 3,
 		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T> }= 4,
+		// Utility: pallet_utility::{Pallet, Call, Event<T>}
```

Comment:
should this get removed if not necessary, usually commented code confuses others